include/picongpu/param/species.param

```diff
@@ -76,6 +76,7 @@ using UsedParticleCurrentSolver = currentSolver::Esirkepov< UsedParticleShape >;
  *                                              with classical radiation reaction
  *
  * For diagnostics & modeling: ------------------------------------------------
+ * - particles::pusher::Acceleration : constant force on particles for acceleration
```

Comment:
I think it is constant field, but not necessarily constant force, as the latter depends on the charge?